src/main/java/org/zalando/nakadi/util/Aruha771MigrationHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.zalando.nakadi.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zalando.nakadi.config.JsonConfig;
+import org.zalando.nakadi.domain.Subscription;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * Temporary class to be used for DB migration of ARUHA-771
+ */
+public class Aruha771MigrationHelper {
```

Comment:
I'm not sure. Let's suppose someone uses Nakadi outside of our stack (I know no one does). This user would be interested in this. Also, it would be useful to have some upgrade instructions for this user to perform while upgrading.

But, yeah, given no one else uses it yet, it would be better to have it outside.